tools/toolchains/arm.py

```diff
@@ -31,12 +31,18 @@
 
 ARMC5_MIGRATION_WARNING = (
     "Warning: We noticed that you are using Arm Compiler 5. "
-    "We are deprecating the use of Arm Compiler 5 soon. "
+    "We are deprecating the use of Arm Compiler 5. "
     "Please upgrade your environment to Arm Compiler 6 "
     "which is free to use with Mbed OS. For more information, "
     "please visit https://os.mbed.com/docs/mbed-os/latest/tools/index.html"
 )
 
+UARM_TOOLCHAIN_WARNING = (
+    "Warning: We noticed that you are using uARM Toolchain either via mbed-cli -t command line or default_toolchain. "
+    "We are deprecating the use of uARM Toolchain. "
+    "Please use bare metal with a small library. For more information, "
+    "please visit https://os.mbed.com/docs/mbed-os/latest/reference/mbed-os-bare-metal.html"
```

Comment:
We should not mix up bare metal and uARM. say: "Warning: We noticed that you are using uARM toolchain. We are deprecating the use of uARM toolchain. Use the C library "small" option with the ARM toolchain. For more information,  please visit xxx"

Reply:
```
Warning: We noticed that you are using uARM toolchain. We are deprecating the uARM toolchain.
Use the ARM toolchain with Mbed OS configuration parameter "target.c_lib" set to "small".
For more information, please visit xxx
```